src/managing-workflow/tuning-component-settings.md

```diff
@@ -39,7 +39,9 @@ RESERVED_NAMES                                  | a comma-separated list of name
 SLUGRUNNER_IMAGE_NAME                           | the image used to run buildpack application slugs (default: "quay.io/deisci/slugrunner:canary")
 SLUG_BUILDER_IMAGE_PULL_POLICY                  | the kubernetes [image pull policy][pull-policy] for slugbuilder (default: "Always")
 DOCKER_BUILDER_IMAGE_PULL_POLICY                | the kubernetes [image pull policy][pull-policy] for dockerbuilder (default: "Always")
-DEIS_DEPLOY_BATCHES                             | the number of pods to bring up and take down sequentially during a scale (default: all)
+DEIS_DEPLOY_BATCHES                             | the number of pods to bring up and take down sequentially during a scale (default: number of available nodes)
+DEIS_KUBERNETES_DEPLOYMENTS                     | if [Deployments][] is used to handle an application deploy instead off [ReplicationControllers][] (default: off)
```

Comment:
The value to enable it would be useful here: on, true, yes, plz, yesplz, omgwtfbbqyes

Should also state that this defines the default platform behavior